dags/map_reproducibility/nemo_gpt3.py

```diff
@@ -73,25 +76,73 @@ def run_aotc_workload():
                 + install_helm_cmds()
                 + namespace_cmds()
                 + workload_cmds
-                + helm_install_cmds()
-                + wait_for_jobs_cmds()
+                # + helm_install_cmds()
```

Comment:
Can you clean up any commented-out code that is unneeded?

Reply:
Done